app/src/main/java/com/example/hangouts/BayesianRating.java

```diff
@@ -24,14 +25,91 @@ public static double getBayesianRating(List<Double> list, double significance) {
         if (sum == 0) {
             return 0;
         }
-        double z_value = getPpf(1 - (1 - significance) / 2);
+        double zValue = getPpf(1 - (1 - significance) / 2);
         double firstTerm = 0;
         double secondTerm = 0;
         for (int i = 0; i < size; i++) {
             firstTerm += (i + 1) * (list.get(i) + 1) / (sum + size);
             secondTerm += (i + 1) * (i + 1) * (list.get(i) + 1) / (sum + size);
         }
-        return firstTerm - z_value *
-                Math.sqrt((secondTerm - firstTerm*firstTerm)/(sum + size + 1));
+        return firstTerm - zValue *
+                Math.sqrt((secondTerm - firstTerm * firstTerm) / (sum + size + 1));
+    }
+
+    private static double inverseCumulativeProbability(double x) {
+        return FastMath.sqrt(2.0) * inverseErrorFunction(2 * x - 1);
+    }
+
+    /***
+     * The algorithm to approximate the error inverse function with the finite series method
+     * would look like this, however not only this is not computationally inefficient,
+     * the precision of the double data type won't be able to handle the calculations after the
+     * third term.
+     * final static double SQRTPI_OVER_TWO = FastMath.sqrt(FastMath.PI) / 2;
+     *  final double epsilon = 0.001;
```

Comment:
are these your words? or copy pasted from another lib?
The explanation is nice, just want to give attribution if needed.

Reply:
Yes, I tries to explain myself why this method would not work with java's native datatypes.

Reply:
but I used this as reference https://dlmf.nist.gov/7.17